configure.ac

```diff
@@ -376,7 +376,7 @@ fi
 if test x$use_sanitizers != x; then
   dnl First check if the compiler accepts flags. If an incompatible pair like
   dnl -fsanitize=address,thread is used here, this check will fail. This will also
-  dnl fail if a bad argument is passed, e.g. -fsanitize=undfeined
+  dnl fail if a bad argument is passed, e.g. -fsanitize=undefined
```

Comment:
Isn't this typo an example of a bad argument?

Reply:
Hi - apologies for making you read the code in this state. Initially I had opened the PR because I had a patch that was working under Linux and hadn't yet discovered the CI tests. Since then I've been pushing sporadically as a sanity check against the CI test behavior on my own machine. I haven't really worked with github before; is there a way to disable review on a PR (but not close the PR itself) until the code is ready?

I'm not sure how those commits got duplicated or how someone else's commits got pulled in. I believe it happened the last time I was attempting to rebase onto master. In any case I do intend to clean everything up once I can get all the tests to pass.

I'll restore the typo; at the time I hadn't realized that "undefined" was a valid argument.

I'm confused by your comment about building under windows, so I think I'm missing some fundamental point about the original issue. I had expected that as part of making the change the fuzz tests would now be built by default in more contexts (ie Windows, and specifically as an example the win64 CI test) and any compilation errors would need to be fixed as part of it. If that's not true, then as alternatives I can imagine (a) explicitly not building under non-compiling environments regardless of the value of ENABLE_FUZZ, or (b) ignoring the fact that the CI tests break (if that is even possible), but neither seemed that desirable? Anyway please let me know what behavior you'd like and I'll make the change.

Reply:
(I suppose a third option would be to require someone building under e.g. windows to specify --disable-fuzz)